document/FileFormat.mdk

```diff
@@ -2602,6 +2602,8 @@ This property should be absent only if the tool does not have physical location
 
 NOTE: This could happen, for example, if this `annotatedCodeLocation` occurs within a binary for which
 the tool does not have associated symbol information.
+It might also happen if the value of the `kind` property (&sect;[#annotatedCodeLocation-kind]) is `"functionExit"`,
```

Comment:
Good point, fixed.

---
In reply to: [77726000](https://github.com/sarif-standard/sarif-spec/pull/264#discussion_r77726000) [](ancestors = 77726000)